Dockerfile

```diff
@@ -195,7 +195,26 @@ RUN rustup component add \
   rust-src \
   rustfmt
 
+# No binary available on Github, have to use cargo install
 RUN cargo install cargo-deadlinks
-RUN cargo install cargo-deny
-RUN cargo install cargo-crev
-RUN cargo install grcov
+
+# Where to install rust tooling
+ARG install_dir=${rustup_dir}/bin
```

Comment:
Maybe we should have an overall `tools_dir` and use it for most single-binary tools?

Reply:
I did initially that. But then I noticed that this is where the tools went initially, and as these were rust based, so it seemed natural.
Usually the tools install in `/usr/local/bin` or `/usr/bin`, but we could have another folder.